src/test/regress/sql/view_propagation.sql

```diff
@@ -356,6 +356,24 @@ ALTER VIEW IF EXISTS non_existing_view RENAME COLUMN val1 TO val2;
 ALTER VIEW IF EXISTS non_existing_view RENAME val2 TO val1;
 ALTER VIEW IF EXISTS non_existing_view SET SCHEMA view_prop_schema;
 
+-- verify that the views get distributed after the table is distributed
```

Comment:
Could you please add tests to show:
- Handling views
    - Depending on a Citus metadata table and dist table
    -  Depending on a Postgres metadata table and dist table
    - Owned by extension
- Adding same set of tests for PG table -> Citus local table
- Adding tests to make sure that we keep views on the worker after Citus local table -> Distributed table

Reply:
> Adding same set of tests for PG table -> Citus local table

I think pg table -> dist/ref is probably the most common scenario, so let's have test for those as well